tests/netgroup/test_netgroup_member.yml

```diff
@@ -0,0 +1,120 @@
+---
+- name: Netgroup member test
+  hosts: "{{ ipa_test_host | default('ipaserver') }}"
+  become: true
+
+  tasks:
+  - name: Get Domain from server name
+    set_fact:
+      ipaserver_domain: "{{ ansible_facts['fqdn'].split('.')[1:] | join ('.') }}"
+    when: ipaserver_domain is not defined
+
+  - name: Set host1_fqdn .. host2_fqdn
+    set_fact:
+      host1_fqdn: "{{ 'host1.' + ipaserver_domain }}"
+      host2_fqdn: "{{ 'host2.' + ipaserver_domain }}"
+
+  # CLEANUP TEST ITEMS
```

Comment:
Please, cleanup test items also after test is executed.

Ideally, use:
```yaml
block:
   # tests
always:
   # cleanup
```
This way the environment is cleaned even if a test fails.